presto-tests/src/main/java/com/facebook/presto/tests/DistributedQueryRunner.java

```diff
@@ -350,7 +350,7 @@ private DistributedQueryRunner(
 
         long start = nanoTime();
         while (!allNodesGloballyVisible()) {
-            Assertions.assertLessThan(nanosSince(start), new Duration(60, SECONDS));
+            Assertions.assertLessThan(nanosSince(start), new Duration(100, SECONDS));
```

Comment:
@elharo Thanks for increasing timeout. However I think the problem will still exisit and I see query runners can't start even with 10 minutes. My guess is something happens to start up the worker, but I am not sure because I was not able to see the logs what's happening with the subprocess. This needs a bit more digging.